main/main.ino

```diff
@@ -1613,6 +1622,65 @@ void receivingMQTT(char* topicOri, char* datacallback) {
   }
 }
 
+#ifdef MQTT_HTTPS_FW_UPDATE
+#  include <WiFiClientSecure.h>
+#  ifdef ESP32
+#    include <HTTPUpdate.h>
+#  elif ESP8266
+#    include <ESP8266httpUpdate.h>
+#  endif
+void MQTTHttpsFWUpdate(char* topicOri, JsonObject& HttpsFwUpdateData) {
+  if (strstr(topicOri, subjectFWUpdate) != NULL) {
+    if (strcmp(HttpsFwUpdateData["version"], OMG_VERSION) != 0) {
```

Comment:
Maybe we could add also a key and a check against the ota_password for security purposes.

Reply:
Ok, will do!